lewis/adapters/epics.py

```
@@ -145,8 +212,9 @@ def process_pv_updates(self, force=False):
 
         self.updatePVs()
```

Do you remember why we silently ignore AttributeError and TypeError just above here?

---

It might have been a "safety net" from very early days when there was a risk that the property did not exist on the adapter. Another thing that can go wrong is that the property getter returns an invalid value type, so that `setParam` might raise one of these exceptions - but I'm not entirely sure about it.